### PR TITLE
chore: update lerna to 4.0.0

### PR DESCRIPTION
**Pull request by armano2**

### CODE REVIEW

**Review by bradzacher — DISMISSED** [review]
did this really generate 3k lines of changes to the yarn.lock? JFC they have too many dependencies.

**Comment by armano2** [comment]
they rely on conventional-chanangleog-parser that requires on meow, you can read about this here:

https://github.com/sindresorhus/meow/issues/177

File: package.json [hunk 1]
```diff
@@ -44,36 +44,12 @@
...
-      ]
-    }
-  },
   "engines": {
     "node": "^10.12.0 || >=12.0.0"
   },
   "devDependencies": {
     "@commitlint/cli": "^12.0.1",
     "@commitlint/config-conventional": "^12.0.1",
-    "@commitlint/config-lerna-scopes": "^12.0.1",
```
Comment (JounQin) [inline package.json:76]: Just interested, what's wrong with `@commitlint/config-lerna-scopes` package?

Comment (armano2) [reply package.json:76]: current version is now working with lerna 4, PR with fix has been merged but not yet released: https://github.com/conventional-changelog/commitlint/pull/2447

**Review by bradzacher — APPROVED** [review]

**Comment by bradzacher** [comment]
just needs rebasing/updating to resolve the merge conflict

**Comment by armano2** [comment]
> 
> 
> just needs rebasing/updating to resolve the merge conflict

done

**Review by bradzacher — DISMISSED** [review]

**Review by bradzacher — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -55,11 +55,12 @@
...
   "devDependencies": {
-    "@babel/code-frame": "^7.14.5",
-    "@babel/parser": "^7.15.7",
-    "@babel/types": "^7.15.6",
+    "@babel/code-frame": "^7.16.0",
+    "@babel/parser": "^7.16.2",
+    "@babel/types": "^7.16.0",
     "@commitlint/cli": "^14.1.0",
     "@commitlint/config-conventional": "^14.1.0",
+    "@commitlint/config-lerna-scopes": "^14.0.0",
```
Comment (armano2) [inline package.json:63]: we don't need this package anymore

**Pull request merged**